chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertificationIdController.java

```diff
@@ -49,7 +51,10 @@
 
     @Autowired
     private CertificationIdManager certificationIdManager;
-    
+
+    @Autowired
+    private FF4j ff4j;
```

Comment:
Can you update this class to use constructor injection?

Reply:
Addressed.